client/components/loading-spinner/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Spinner from 'components/spinner';
+
+const LoadingSpinner = ( { inline } ) => {
```

Comment:
We can remove the inline loading spinner and use the [busy button component from Calypso](https://wpcalypso.wordpress.com/devdocs/design/button-group).

If we do that, we can remove `<LoadingSpinner />` entirely from our codebase.